zoo-css-challenge/style.css

```
@@ -178,6 +190,8 @@ footer {
 .membership {
   text-align: center;
   background-color: #fffffc;
+  max-width: 700px;
+  padding: 50px;
```

```suggestion
  margin: auto;
```
We can use `margin: auto` to align the element in the middle. `auto` distributes the margin evenly on the left and right sides